css-two-column-layout/styles.css

```diff
@@ -39,60 +39,53 @@ h4 {
 .paragraph {
   color: gray;
   text-align: start;
-  font-size: x-small;
+  font-size: 10.5px;
   line-height: 1.2rem;
 }
 
 .row {
   display: flex;
-  width: 90%;
   flex-wrap: wrap;
-  justify-content: space-evenly;
-  margin-top: 30px;
-  margin-bottom: 0;
-  margin-left: auto;
-  margin-right: auto;
 }
 
 .column-half {
-  display: flex;
   width: 50%;
-  margin-bottom: 25px;
-  padding-right: 5%;
-  padding-left: 5%;
+  display: flex;
+  padding-top: 20px;
+  padding-bottom: 20px;
+  padding-right: 7%;
+  padding-left: 7%;
+  position: relative;
```

Comment:
This is still a bunch of excess styling on your `column` class. COlumn classes should only include a `width` or `flex-basis`. You should make an additional class you can add along with the `column-half` class.